README.md

```diff
@@ -61,13 +60,19 @@ end
 ```
 
 </td>
-<td>
+</tr>
+<tr>
+  <th colspan="2"> PowerShell </th>
+</tr>
+<tr>
+<td colspan="2">
 
 ```powershell
 function ll() {
-  cd "$(llama $args)"
+  cd $(llama $args | Out-String -Stream | Select-Object -Last 1)
 }
 ```
+See [issues/30](https://github.com/antonmedv/llama/issues/30) for more details.
```

Comment:
Still in pr

Reply:
Yeah, I kept that in the actual README so that when users read the setup docs they can understand that the script is actually a workaround for an upstream issue (all that info is in #30), that's something that could be removed once the issue is closed and the script is updated again. I just removed the association in the PR text that would close the issue once it's merged.